highfidelity/hifi
# Add eslintrc file

Added an eslint config file to the root of `scripts/` directory.  It is configured to enforce the official coding standard.

To Test:

1. [Install][integrations] the eslint plugin for your favorite text editor
2. Open a script file, it should highlight code style violations

**OR**

1. [Install][cli] the command line interface
2. Run `eslint -c .eslintrc.js --color path/to/your/script.js` from the `scripts/` directory, replacing `path/to/your/script.js` with any script you suspect of violating the coding standard.

[integrations]: http://eslint.org/docs/user-guide/integrations
[cli]: http://eslint.org/docs/user-guide/command-line-interface
